.travis.yml

```diff
@@ -24,11 +24,10 @@ install:
 script:
     - pytest tests
 deploy:
-    - provider: pypi
-      skip_cleanup: true
-      user: datamade.wheelbuilder
-      on:
+    provider: pypi
+    user: datamade.wheelbuilder
+    on:
         tags: true
-        condition: "$TRAVIS_OS_NAME = osx"
```

Comment:
Since I'we're running multiple builds to test different Python versions, don't we only want to limit the deploy to one build?